.coveragerc

```diff
@@ -625,6 +625,7 @@ omit =
     homeassistant/components/media_player/yamaha.py
     homeassistant/components/media_player/ziggo_mediabox_xl.py
     homeassistant/components/mycroft.py
+    homeassistant/components/mythicbeastsdns.py    
```

Comment:
We could remove this, since we have good coverage with the tests.

Reply:
Done :)